.gitignore

```diff
@@ -3,3 +3,4 @@ target
 helix-term/rustfmt.toml
 result
 runtime/grammars
+runtime
```

Comment:
This change is unrelated and should be removed. The runtime files are part of the repo and should not be gitignored

Reply:
How do I undo this commit? Should I just delete that in a new commit have it be squashed later?

Reply:
We always squash snakmer ORs kune this anyway so don't worry about it

Reply:
> We always squash snakmer ORs kune this anyway

hmm??